cmd/system-shutdown/system-shutdown.c

```diff
@@ -93,7 +101,19 @@ int main(int argc, char *argv[])
 		}
 	}
 
-	reboot(cmd);
+	// glibc reboot wrapper does not expose the optional reboot syscall
+	// parameter
```

Comment:
you could also use `reboot(2)` from `linux/reboot.h` instead of `sys/reboot.h`, but not both of these at the same time.